components/script/webdriver_handlers.rs

```diff
@@ -185,6 +180,44 @@ pub fn handle_get_title(page: &Rc<Page>, _pipeline: PipelineId, reply: IpcSender
     reply.send(String::from(page.document().Title())).unwrap();
 }
 
+pub fn handle_get_rect(page: &Rc<Page>,
+                       pipeline: PipelineId,
+                       element_id: String,
+                       reply: IpcSender<Result<Rect<f64>, ()>>) {
+    reply.send(match find_node_by_unique_id(&*page, pipeline, element_id) {
+        Some(elem) => {
+            // https://w3c.github.io/webdriver/webdriver-spec.html#dfn-calculate-the-absolute-position
+            match elem.downcast::<HTMLElement>() {
+                Some(html_elem) => {
+                    // Step 1
+                    let mut x: i32 = 0;
+                    let mut y: i32 = 0;
```

Comment:
nit: Instead of specifying the type of x and y, specify the type of the 0 constant instead (i.e. use `0i32` instead of 0).